MAUI/Maps/migration.md

```diff
@@ -363,47 +363,47 @@ The following code example explains how to intialize the layer and customize the
    </td>
    <td>
-      Can achieve all this using {{'[UrlTemplate]()'| markdownify }} and {{'[GetBingUrl()]()'| markdownify }}
+      Can achieve all this using {{'[UrlTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapTileLayer.html#Syncfusion_Maui_Maps_MapTileLayer_UrlTemplate)'| markdownify }} and {{'[GetBingUrl()]()'| markdownify }}
    </td>
 </tr>
 <tr>
    <td>
       {{'[GeoCoordinates](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html#Syncfusion_SfMaps_XForms_ImageryLayer_GeoCoordinates)'| markdownify }}
    </td>
    <td>
-      {{'[*Center]()'| markdownify }} 
+      {{'[*Center](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapTileLayer.html#Syncfusion_Maui_Maps_MapTileLayer_Center)'| markdownify }} 
    </td>
 </tr>
 <tr>
    <td>
       {{'[CanCacheTiles](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html#Syncfusion_SfMaps_XForms_ImageryLayer_CanCacheTiles)'| markdownify }}
    </td>
    <td>
-      {{'[CanCacheTiles]()'| markdownify }}
+      {{'[CanCacheTiles](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapTileLayer.html#Syncfusion_Maui_Maps_MapTileLayer_CanCacheTiles)'| markdownify }}
    </td>
 </tr>
 <tr>
    <td>
       {{'[Radius](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html#Syncfusion_SfMaps_XForms_ImageryLayer_Radius)'| markdownify }} 
    </td>
    <td>
-      {{'[Radius]()'| markdownify }} 
+      {{'[Radius](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapTileLayer.html#Syncfusion_Maui_Maps_MapTileLayer_Radius)'| markdownify }} 
    </td>
 </tr>
 <tr>
    <td>
       {{'[DistanceType](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html#Syncfusion_SfMaps_XForms_ImageryLayer_DistanceType)'| markdownify }} 
    </td>
    <td>
-      {{'[DistanceType]()'| markdownify }}
+      {{'[DistanceType](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapTileLayer.html#Syncfusion_Maui_Maps_MapTileLayer_DistanceType)'| markdownify }}
    </td>
 </tr>
 <tr>
    <td>
       {{'[LatLngBounds](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html#Syncfusion_SfMaps_XForms_ImageryLayer_LatLngBounds)'| markdownify }} 
    </td>
    <td>
-      {{'[*MapLatLngBounds]()'| markdownify }} 
+      {{'[*MapLatLngBounds](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapTileLayer.html#Syncfusion_Maui_Maps_MapTileLayer_MapLatLngBounds)'| markdownify }} 
```

Comment:
As of now, just change the property to `LatLngBounds`. We can change the API link later.

Reply:
Changed as `LatLngBounds`